packages/documentation/static/code-runtime/vue/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ix-vue-example",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "vue-tsc --noEmit && vite build",
+    "serve": "vite preview"
+  },
+  "dependencies": {
+    "vue": "^3.2.6",
+    "@siemens/ix": "^1",
+    "@siemens/ix-icons": "^1",
+    "@siemens/ix-vue": "^1"
+  },
+  "devDependencies": {
+    "@vitejs/plugin-vue": "^1.6.1",
+    "@vue/compiler-sfc": "^3.2.6",
+    "typescript": "^4.3.2",
+    "vite": "^2.5.4",
```

Comment:
We should at least align the major version of vite with the other versions inside the repo. `vite with major version 3`

Reply:
there is already version 4, we could go straight to that one and also update the remaining repo dependencies

Reply:
Also great you can do this inside this PR or in a separate one.